plugins/modules/keycloak_user_federation.py

```diff
@@ -843,6 +864,7 @@ def main():
 
     if mappers is not None:
         for mapper in mappers:
+            set_mapper_defaults(mapper)
```

Comment:
This is likely a breaking change (as above), since it sets values that the user did not provide.

It's OK to set them when calling the API to create something, but not before doing idempotency checks.